src/status_im/wallet/core.cljs

```diff
@@ -28,21 +28,13 @@
             [status-im.ui.components.bottom-sheet.core :as bottom-sheet]))
 
 (defn get-balance
-  [{:keys [address on-success on-error number-of-retries]
-    :as params
-    :or {number-of-retries 4}}]
-  (log/debug "[wallet] get-balance"
```

Comment:
Do we have other logs instead? Previous time it helped to spot an issue in @Ferossgp local changes, so I would keep them as is

Reply:
ok i'll add them, thanks